src/scenarios/Slider.scenario.tsx

```diff
@@ -0,0 +1,124 @@
+import { Box, Slider, Typography, TextField } from '@mui/material'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Layout from '../components/Layout'
+import PageSetup from '../components/PageSetup'
+
+const BasicSlider: React.FC = () => {
+  const { t } = useTranslation()
+  const [basicValue, setBasicValue] = useState<number>(0)
+
+  const handleBasicChange = (_: Event, newValue: number | number[]): void => {
+    setBasicValue(newValue as number)
+  }
+
+  return (
+    <Box>
+      <Typography gutterBottom>{t('scenarios.slider.basic-slider')}</Typography>
+      <Slider
+        value={basicValue}
+        onChange={(_, newValue) => {
+          handleBasicChange(_, newValue as number)
+        }}
+        valueLabelDisplay='auto'
+        min={1}
+        max={100}
+        data-testid='basic-slider'
+      />
+    </Box>
+  )
+}
+
+const RangeSlider: React.FC = () => {
+  const { t } = useTranslation()
+
+  const [rangeValue, setRangeValue] = useState<number[]>([0, 25])
+  const handleRangeChange = (_: Event, newValue: number | number[]): void => {
+    setRangeValue(newValue as number[])
+  }
+
+  return (
+    <Box>
+      <Typography gutterBottom>{t('scenarios.slider.range-slider')}</Typography>
+      <Slider
+        value={rangeValue}
+        onChange={(_, newValue) => {
+          handleRangeChange(_, newValue as number[])
+        }}
+        valueLabelDisplay='auto'
+        min={1}
+        max={100}
+        disableSwap={true}
+        data-testid='range-slider'
+      />
+    </Box>
+  )
+}
+
+const SliderWithInput: React.FC = () => {
+  const { t } = useTranslation()
+
+  const [inputValue, setInputValue] = useState<number>(0)
+
+  const handleSliderChange = (_: Event, newValue: number | number[]): void => {
+    setInputValue(newValue as number)
+  }
+
+  const handleInputChange: React.ChangeEventHandler<HTMLInputElement> = (
+    event
+  ) => {
+    const newValue = Number(event.target.value)
+    setInputValue((prevValue) => {
+      const clampedValue = Math.min(100, Math.max(1, newValue))
+      return isNaN(clampedValue) ? prevValue : clampedValue
+    })
+  }
+
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center', mt: 2 }}>
```

Comment:
remove 'display' and 'alignItems'
Add Slider and TextField into a Grid

This will align the content like the others: title on top of the slider